src/desktop/apps/collect/server.tsx

```diff
@@ -22,7 +22,9 @@ const index = async (req, res, next) => {
       routes: collectRoutes,
       url: req.url,
       userAgent: req.header("User-Agent"),
-      context: buildServerAppContext(req, res, { COLLECTION_HUBS }),
+      context: buildServerAppContext(req, res, {
+        COLLECTION_HUBS,
+      }),
```

Comment:
Pass A/B test ENV var on server-side pass.

Reply:
this one just looks like a formatting change?

Reply:
Yah the only real change is in `client.tsx`